frontend/components/MainFlow.vue

```diff
@@ -70,7 +74,15 @@ export default Vue.extend({
             type: Array as Vue.PropType<AuctionTransaction[]>,
             default: () => [],
         },
-        isAuctionsLoading: {
+        takeEventStorage: {
+            type: Object as Vue.PropType<string, TakeEvent[]>,
+            default: () => {},
```

Comment:
This will result in `undefined`. Have you meant that?
```suggestion
            default: () => ({}),
```